pnpm-lock.yaml

```diff
@@ -196,7 +196,7 @@ importers:
       '@sveltejs/adapter-cloudflare-workers': link:../../../adapter-cloudflare-workers
       '@sveltejs/adapter-netlify': link:../../../adapter-netlify
       '@sveltejs/adapter-vercel': link:../../../adapter-vercel
-      '@sveltejs/kit': 1.0.0-next.84_svelte@3.37.0+vite@2.1.5
+      '@sveltejs/kit': link:../../../kit
```

Comment:
There probably shouldn't be any changes to this file since `package.json` wasn't edited. This looks like leftover from testing

Reply:
it looks to me like the new version is correct